tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist/",
+    "sourceMap": true,
+    "strictNullChecks": true,
+    "module": "es6",
+    "lib": ["es2020", "dom"],
+    "jsx": "react",
+    "target": "es5",
+    "allowJs": false,
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "declaration": true
```

Comment:
I think we also want `"strict": true`, no?

What about `"noImplicitAny": true`? Does that make some things more difficult?